src/pages/HomePage.js

```diff
@@ -1,16 +1,21 @@
-import React from "react";
-import MostPopularServices from "../components/MostPopularServices";
-
+import React from 'react';
+import MostPopularServices from '../components/MostPopularServices';
+import NewsRoom from '../components/NewsRoom';
 const HomePage = () => (
-  <div className="dg_section dark">
-    <div className="container">
-      <div className="row">
-        <div className="col">
-          <MostPopularServices />
-        </div>
-      </div>
-    </div>
-  </div>
+	<div className="dg_section dark">
+		<div className="container">
+			<div className="row">
+				<div className="col">
+					<MostPopularServices />
+				</div>
+			</div>
+			<div className="row">
+				<div className="col">
+					<NewsRoom />
```

Comment:
See @tmccoy529 example for how we are handling sections, this was jsut decided this morning, but will be easier to make the changes.

Section - https://github.com/baltimorecounty/dotgov-prototype-2.0/blob/SectionYourGovernment/src/components/sections/YourGovernmentSection.js

Hompage - https://github.com/baltimorecounty/dotgov-prototype-2.0/blob/SectionYourGovernment/src/pages/HomePage.js

If you want to make this easier you can merge that branch "SectionYourGovernemnt" as it hasn't yet made it into integration, but will soon.

Please let me know if you need any help with this.